post-processor/vagrant/libvirt_test.go

```diff
@@ -70,3 +74,76 @@ func Test_sizeInMegabytes_WithExabytesUnit(t *testing.T) {
 	assertSizeInMegabytes(t, "1234E", 1234*1024*1024*1024*1024)
 	assertSizeInMegabytes(t, "1E", 1*1024*1024*1024*1024)
 }
+
+func Test_ManyFilesInArtifact(t *testing.T) {
+	p := new(LibVirtProvider)
+	ui := testUi()
+	type testCases struct {
+		Files         []string
+		Format        string
+		FilesExpected []string
+	}
+	testcases := []testCases{
+		{
+			[]string{},
+			"qcow2",
+			[]string{},
+		},
+		{
+			[]string{"test"},
+			"vmdk",
+			[]string{"box_0.img"},
+		},
+		{
+			[]string{"test", "/test-1", "test-2"},
+			"qcow2",
+			[]string{"box_0.img", "box_1.img", "box_2.img"},
+		},
+	}
+	for _, tc := range testcases {
+		dir, _ := tmp.Dir("pkr")
+		defer os.RemoveAll(dir)
+
+		actifactFiles := []string{}
+		for _, file := range tc.Files {
+			fullFilePath := fmt.Sprintf("%s/%s", dir, file)
+			actifactFiles = append(actifactFiles, fullFilePath)
+			_, err := os.Create(fullFilePath)
+			if err != nil {
+				t.Fatalf("Can't create %s : %s", fullFilePath, err)
+			}
+		}
+
+		artifact := &packersdk.MockArtifact{
+			FilesValue: actifactFiles,
+			StateValues: map[string]interface{}{
+				"diskType":   tc.Format,
+				"diskSize":   "1234M",
+				"diskName":   "test",
+				"domainType": "kvm",
+			},
+		}
+
+		dir_process, _ := tmp.Dir("process")
```

Comment:
Nit: in Go snake_case is not really idiomatic, in general camelCase is preferred.

```suggestion
		dirProcess, _ := tmp.Dir("process")
```